packages/gatsby/package.json

```diff
@@ -88,6 +88,7 @@
     "gatsby-react-router-scroll": "^4.3.0-next.0",
     "gatsby-telemetry": "^2.3.0-next.0",
     "glob": "^7.1.6",
+    "gatsby-plugin-functions": "^0.1.0-0",
```

Comment:
Is adding it here necessary? I don't see the other internal-plugins inside this package.json

Reply:
Yeah, if this is experimental - maybe we can just add it to "installation instructions" for now?

Reply:
Oh, it's actually another package 😮 So is this published from another branch then? Then it definitely shouldn't be part of the general package yet

Reply:
This is left over from before it got moved to internal-plugins. Removing it now.